packages/react-native-payments/lib/js/PaymentRequest.js

```diff
@@ -403,10 +403,10 @@ export default class PaymentRequest {
     return this._acceptPromiseRejecter(new GatewayError(details.error));
   }
 
-  _closePaymentRequest() {
+  _closePaymentRequest(reject = true) {
     this._state = 'closed';
 
-    this._acceptPromiseRejecter(new Error('AbortError'));
+    if (reject) this._acceptPromiseRejecter(new Error('AbortError'));
```

Comment:
maybe more bulletproof:
```suggestion
    if (this._acceptPromiseRejecter) this._acceptPromiseRejecter(new Error('AbortError'));
```

Reply:
or do u want to avoid rejecting from stopRequest?

Reply:
> maybe more bulletproof:

Yeah, we could do that from the start. But we might miss some actual bugs/edge cases then

Reply:
> or do u want to avoid rejecting from stopRequest?

That was the only goal here